Parent/Parent/Details/AnnouncementDetailsViewController.swift

```diff
@@ -111,4 +111,9 @@ class AnnouncementDetailsViewController: DiscussionTopic.DetailViewController {
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        navigationController?.isNavigationBarHidden = false
```

Comment:
What in the world. Why are these needed?

Reply:
No idea. I couldn't figure that out. But without it the nav bar was hidden.

Reply:
Resulting in no way of getting away from the screen.